### ISSUE TITLE
This plugin doesn't work..

### ISSUE THREAD

**Issue by Benjade** [rainlab/translate-plugin#751]
The plugin doesn't work, I spent 1 hour searching, impossible to make it work and I'm not that beginner.
So I installed the plugin, changed nothing, I added a FR language, I went to Pages, changed the language to FR, modified the content to FR, saved then I went to the site, I changed the language to FR of the translated page, and I have the nice error: Page not found
We're sorry, but the page you requested cannot be found.

After that I searched for 1 hour what it could be, trying several things but nothing works.

I put https://mysite.com/fr for the FR language in Site definitions > Config. I also tried without it.. 

In short, it's really tedious.

**Comment by daftspunk** [comment]
Hey @Benjade 

Thanks for sharing your experience, and sorry to hear about your frustration with this plugin. This plugin used to serve as the main approach for multi lingual sites for October CMS. More recently, there is a newer feature called multi site. Both have distinct approaches to translation, and both serve a purpose. 

The approach used by this plugin is a little more complicated, so we recommend you take a look at the following tutorial, it could be more appropriate for your use case, and could be a better place to start:

- **How to create multilingual sites with October CMS Multisite**
   https://www.youtube.com/watch?v=_kX7P3SEHg8

I hope this helps.

**Comment by Benjade** [comment]
I had already tried something similar, and it didn't work either. I reinstalled everything, including October, and now it's working.

Thank you, daftspunk :)

**Comment by Benjade** [comment]
And the submenu does not work with the translation, there is no /fr

![stats1](https://github.com/user-attachments/assets/c2ae666b-4135-4cb7-bde8-65cf65075a66)
![stats2](https://github.com/user-attachments/assets/d1c5c17c-b5d0-4327-9cad-1230f08123b2)

**Comment by Benjade** [comment]
It seems that when editing in the editor, it translates the 2 languages ​​simultaneously, in Pages it works well, but it is limited since not all the files are there.

**Comment by Benjade** [comment]
Also, if I create a Maintenance page in Pages, it doesn't show up in the Maintenance section's page list. It only shows up if I create it from the editor.. Seems very buggy, at least unusable for now.

**Comment by daftspunk** [comment]
Hmmm, there is a gap in the Bonjour theme or the Pages plugin where the multisite context is not applied to the `page.url` property. 

For new sites, it might be best to look at the "Wiki / Article" example in the Demo theme that ships with October CMS, it uses Tailor, which is a newer tool and could serve as a substitute for the Pages plugin. A lot of this stuff has been improved / fixed there...

I hope this helps!
